doc/protocol/support.md

```diff
@@ -1,68 +1,87 @@
-| Read                          |            x/o                ||
-|   - Authorized Read           |             x                 | Authorization must be performed as standalone call, do not support "in-lining" authorization in read call         |
-|   - Search Read               |             o                 ||
-|   - History Read              |             o                 ||
-|   - Signal Discovery Read     |             o                 ||
-|   - Dynamic Metadata Read     |             o                 ||
-| Update                        |             x                 ||
-|   - Authorized Update         |             x                 | Authorization must be performed as standalone call, do not support "in-lining" authorization in update call       |
-| Subscribe                     |             x                 ||
-|   - Authorized Subscribe      |             x                 | Authorization must be performed as standalone call, do not support "in-lining" authorization in subscribe call    |
-|   - Curve Logging Subscribe   |             o                 ||
-|   - Range Subscribe           |             o                 ||
-|   - Change Subscribe          |             o                 ||
-| Unsubscribe                   |             x                 ||
-| Subscription                  |             x                 ||
-| Error messages                |             x                 ||
-| Timestamps                    |             x                 ||
-
-x = implemented; x/o = partly implemented; o = not implemented
-
-For a more detailed view of the supported JSON-schemas [click here](https://github.com/eclipse/kuksa.val/blob/master/kuksa-val-server/include/VSSRequestJsonSchema.hpp)
-
-
-### KUKSA.val databroker gRPC API
-The VISS Standard is not applicable for gRPC protocol. Here is an overview what the gRPC API in KUKSA.val databroker is capable of:
-
-  * Read: Reading VSS datapoints 
-    * Reading current or target value for actuators
-    * Reading some metadata information from VSS datapoints
-  * Write: Writing VSS datapoints
-    * Writing sensor values
-    * Writing current or target value for actuators
-    * Soon: Writing some metadata information from VSS datapoints
-  * Subscription: Subscribing VSS datapoints
-    * Subscribing sensor values
-    * Subscribing current or target value for actuators
-
-
+## Supported protocols
+
+This file contains an overview what the KUKSA.val Server and databroker each supports. It focuses on gRPC and VISS support and also what feeders are supported.
+
+
+| Protocol                  | KUKSA.val server | KUKSA.val databroker |
+|---------------------------|:----------------:|:--------------------:|
+| VISS V1                   |         -        |           -          |
+| VISS V2                   |        x/-       |          x/-         |
```

Comment:
Isn't it a bit confusing that we in kuksa-client code specifies the two subprotocols:

`subprotocols = ["VISSv2", "VISSv1"]`

but here we say that both server and databroker supports (parts of) VISSv2

Reply:
Would it be better to call them (in kuksa-client) something like "VISSv2" and "VISSv2-extended" or something similar.

Reply:
Yes, that is confusing.

When I added it, I assumed that the "not VISSv2" that kuksa-val-server and kuksa-client speaks was VISSv1. So if any server were to return VISSv1, kuksa-client could just use the that.

But reading the documentation here, it seems that it is actually a modified version of VISSv2 they use. Which is also confusing, because I don't think a VISSv2 would work with kuksa-val-server.

Reply:
Pushed an update which removed `"VISSv1"` from the requested subprotocols.

Reply:
> Would it be better to call them (in kuksa-client) something like "VISSv2-std" and "VISSv2-extended" or something similar.

The VISSv2 [specification](https://www.w3.org/TR/viss2-transport/#wss-initialisation) explicitly states that it SHALL be "VISSv2", so no.
> The sub-protocol name SHALL be 'VISSv2' with the digit 2 being the version number.

I guess support for negotiating subprotocol can be added to kuksa-val-server, and that it then would be possible for kuksa-client to identify a server "speaking kuksa-val-server protocol". But I don't know the effort required to do that.